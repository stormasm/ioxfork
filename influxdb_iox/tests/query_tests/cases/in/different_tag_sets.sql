-- IOX_SETUP: OneMeasurementTwoChunksDifferentTagSet

-- IOX_COMPARE: sorted
SELECT temp FROM h2o;

-- IOX_COMPARE: sorted
SELECT * FROM h2o;
