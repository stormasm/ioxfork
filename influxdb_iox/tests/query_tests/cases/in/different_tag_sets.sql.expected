-- Test Setup: OneMeasurementTwoChunksDifferentTagSet
-- SQL: SELECT temp FROM h2o;
-- Results After Sorting
+------+
| temp |
+------+
| 53.4 |
| 70.4 |
|      |
|      |
+------+
-- SQL: SELECT * FROM h2o;
-- Results After Sorting
+--------+------------+---------+-------+------+--------------------------------+
| city   | other_temp | reading | state | temp | time                           |
+--------+------------+---------+-------+------+--------------------------------+
| Boston | 72.4       |         |       |      | 1970-01-01T00:00:00.000000350Z |
| Boston |            | 51.0    |       | 53.4 | 1970-01-01T00:00:00.000000050Z |
|        | 70.4       |         | MA    |      | 1970-01-01T00:00:00.000000250Z |
|        |            |         | MA    | 70.4 | 1970-01-01T00:00:00.000000050Z |
+--------+------------+---------+-------+------+--------------------------------+